azavea/raster-foundry
# [Experiment] Add a blocking thread pool for blocking Futures

## Overview

- [x] Remove useless `implicit sceneIds: Set[UUID]` params (added a special function to get attr store without zooms, created a separate cached for it, in order not to break approach).
- [x] Remove all `blocking {}` function calls
- [x] Remove global execution context usages

Reason: http://doc.akka.io/docs/akka-http/current/scala/http/handling-blocking-operations-in-akka-http-routes.html

## Testing instructions

Visit projects list page (there should be more than one projects for better experience), after that go on a project page, tiles should load without any delay.
